CODE_OF_CONDUCT.md

```diff
@@ -115,14 +115,14 @@ the community.
 ## Attribution
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage],
-version 2.0, available at
-https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+version 2.0, available [here].
```

Comment:
"here" is not a reasonable name for an out-of-band link like this

Reply:
Done